app/imports/startup/server/profile.js

```diff
@@ -8,7 +8,11 @@ import { Profiles } from '../../api/profile/profile';
 /** Initialize the database with a default data document. */
 function addData(data) {
   console.log(`  Adding: ${data.lastName}`);
-  Profiles.insert(data);
+  const profile = data;
+  if (!profile.socials) {
```

Comment:
`profile.socials = profile.socials || [];`

i think this will do the same thing

Reply:
changed it.